.env.nanobox

```diff
@@ -183,6 +183,10 @@ SMTP_FROM_ADDRESS=notifications@${APP_NAME}.nanoapp.io
 # LDAP_BIND_DN=
 # LDAP_PASSWORD=
 # LDAP_UID=cn
+# tootsuite/mastodon#12021: Convert LDAP username to valid format
```

Comment:
I would remove the repeating comments referencing the PR, nothing else does that

Reply:
Removed all comments referencing the PR